detox/src/devices/drivers/android/exec/ADB.js

```diff
@@ -277,7 +277,7 @@ class ADB {
     const retries = _.get(options, 'retries', 1);
     _.unset(options, 'retries');
 
-    return execWithRetriesAndLogs(cmd, options, undefined, retries);
+    return execWithRetriesAndLogs(cmd, options, retries);
```

Comment:
You know, since the function is called "exec with **retries** and **logs**", and you have put the "logs" inside the options, maybe just put "retries" into the options as well, so we don't have to return to it anymore.

Reply:
I was considering that, myself, and couldn't clearly decide. Perhaps you're right, although there was something about the explicitness of the retires/interval args. wdyt?

Reply:
ok, done. tell me what you think!

Reply:
I bet it took some sweat, but it looks totally worth it. Kudos! Great job 👍👍